tools/topology/topology2/cavs-nocodec-bt.conf

```diff
@@ -77,12 +80,21 @@ Object.Dai {
 		sample_bits		32
 		quirks			"lbm_mode"
 		io_clk		$MCLK
+		version         $SSP_BLOB_VERSION
```

Comment:
can this blob version be added automagically since we know which platform is used?